core/eolearn/tests/test_utils/test_testing.py

```diff
@@ -0,0 +1,58 @@
+import datetime as dt
+
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+from sentinelhub import CRS, BBox
+
+from eolearn.core.constants import FeatureType
+from eolearn.core.types import FeatureSpec, FeaturesSpecification
+from eolearn.core.utils.testing import PatchGeneratorConfig, generate_eopatch
+
+
+def test_generate_eopatch_set_bbox_timestamps() -> None:
+    bbox = BBox((0, 0, 10, 10), crs=CRS("EPSG:32633"))
+    timestamps = [dt.datetime(2019, 1, 1)]
+    patch = generate_eopatch((FeatureType.DATA, "bands"), bbox=bbox, timestamps=timestamps)
+
+    assert patch.bbox == bbox
+    assert patch.timestamps == timestamps
+
+    assert patch[(FeatureType.DATA, "bands")].shape[0] == len(timestamps)
```

Comment:
might check the spatial dimensions as well, since the info is all there. 

Added slight improvements:

```suggestion
def test_generate_eopatch_set_bbox_timestamps() -> None:
    feature = (FeatureType.DATA, "bands")
    bbox = BBox((0, 0, 10, 10), crs=CRS("EPSG:32633"))
    timestamps = [dt.datetime(2019, 1, 1)]
    
    default_config = PatchGeneratorConfig()
    patch = generate_eopatch((FeatureType.DATA, "bands"), bbox=bbox, timestamps=timestamps, config=default_config)

    assert patch.bbox == bbox
    assert patch.timestamps == timestamps

    time, *raster_shape, _ = patch[feature].shape
    assert time == len(timestamps)
    assert raster_shape == default_config.raster_shape
```

Reply:
I suggest the above suggestion is done in a separate test. Let's not test too many things at once.